libpod/config/config.go

```diff
@@ -545,5 +545,6 @@ func (c *Config) checkCgroupsAndAdjustConfig() {
 		logrus.Warningf("Alternatively, you can enable lingering with: `loginctl enable-linger %d` (possibly as root)", rootless.GetRootlessUID())
 		logrus.Warningf("Falling back to --cgroup-manager=cgroupfs")
```

Comment:
This warning should be updated to say faling back to --events-logger=file

Reply:
done